### PR TITLE
refactor(egen): Refactored text embedding new api code according to the template guidelines

### PR DESCRIPTION
**Pull request by rohith-egen**
Refactored all the cells according to the template notebook
<br>
List of updates:
1. Steps are re-organized based on the [new template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb).
2. Renamed REGION as LOCATION
3. Removed the before you begin section as per the new template
4. Grouped Vertex AI SDK initiation with Google cloud project setup steps
5. Included colab enterprise link 
6. Used generalized code for "colab only" cells
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/generative_ai/text_embedding_new_api.ipynb [hunk 1]
```diff
@@ -32,38 +32,29 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/text_embedding_new_api.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fgenerative_ai%2Ftext_embedding_new_api.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/generative_ai/text_embedding_new_api.ipynb:42]: change colab enterprise logo url to:

- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/generative_ai/text_embedding_new_api.ipynb [hunk 2]
```diff
@@ -118,204 +109,147 @@
...
-      "execution_count": null,
-      "metadata": {
-        "id": "2b4ef9b72d43"
+        "id": "8925ff9e165e"
       },
-      "outputs": [],
       "source": [
-        "# Install Vertex AI SDK for Python\n",
-        "!pip3 install --quiet --upgrade google-cloud-aiplatform"
+        "## Getting Started"
```
Comment (kittyabs) [inline notebooks/official/generative_ai/text_embedding_new_api.ipynb:115]: change to "Get started"

File: notebooks/official/generative_ai/text_embedding_new_api.ipynb [hunk 3]
```diff
@@ -118,204 +109,147 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "58707a750154"
+        "id": "6dac73aa3f75"
       },
       "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Install Vertex AI SDK and other required packages"
```
Comment (kittyabs) [inline notebooks/official/generative_ai/text_embedding_new_api.ipynb:124]: change to:

- Install Vertex AI SDK for Python ...

File: notebooks/official/generative_ai/text_embedding_new_api.ipynb [hunk 4]
```diff
@@ -118,204 +109,147 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "74ccc9e52986"
+        "id": "7b49231643e4"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/generative_ai/text_embedding_new_api.ipynb:175]: Remove "Please" in accordance with style guide, and use contraction for "it is"

- Wait until it's finished

File: notebooks/official/generative_ai/text_embedding_new_api.ipynb [hunk 5]
```diff
@@ -118,204 +109,147 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ef21552ccea8"
+        "id": "e487ec618b5e"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "### Set Google Cloud project information and initialize Vertex AI SDK\n",
```
Comment (kittyabs) [inline notebooks/official/generative_ai/text_embedding_new_api.ipynb:213]: Vertex AI SDK for Python

**Review by katiemn — APPROVED** [review]

**Pull request merged**